fastly/block_fastly_service_director.go

```diff
@@ -297,11 +272,11 @@ func flattenDirectors(directorList []*gofastly.Director, directorBackendList []*
 			"retries": int(d.Retries),
 		}
 
+		// NOTE: schema.NewSet expects slice of empty interface so we have to build
+		// this from the Dictionary's Backend field.
 		var b []interface{}
-		for _, db := range directorBackendList {
```

Comment:
Now instead of looping over a list of backends that came from a service backend API call and were subsequently filtered down, we're looping over the backends that were exposed via the Director itself.